test/unit/fetchers/github_test.rb

```diff
@@ -0,0 +1,55 @@
+# encoding: utf-8
+
+require 'helper'
+require 'uri'
+
+describe Fetchers::Github do
+  let(:fetcher) { Fetchers::Github }
+  let(:ref_for_test) { "0ca497e333a3222e4b517bc3ebe43eba01986c4c" }
+  let(:tag_ref_for_test) { "202d4e0f97b5dc2b88df13d686eae36320a5a53c" }
+  let(:uri_without_spec) { URI.parse("github://chef/inspec") }
+  let(:uri_with_branch) { URI.parse("github://chef/inspec/branch/master") }
```

Comment:
I would love to support the existing github fetcher behavior as well, where I am able to just copy the url from the browser